.github/workflows/check.yml

```diff
@@ -0,0 +1,32 @@
+name: "Code check"
+
+on:
+  workflow_call:
+
+  pull_request:
+    paths:
+      - "src/**"
+      - "tests/**"
+      - "composer.json"
+      - "ecs.php"
+      - "rector.php"
+      - "phpstan.neon"
+      - "phpunit.xml"
+
+concurrency:
+  group: larastrict-check-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  code:
+    name: "Code check"
+    uses: wrk-flow/reusable-workflows/.github/workflows/php-check.yml@b0886c7fa81dab2fb2615c06eb66e94711652056
```

Comment:
Tady je záměrně  reference na konkrétní verzi?

Reply:
Jj radši jsem to tam dal ať je to safe do budoucnu. Ale klidně nemusíš